pkg/web/routes.go

```diff
@@ -90,6 +90,7 @@ func (s *Server) setupRoutes() (err error) {
 	s.router.GET("/login", s.LoginPage)
 	s.router.GET("/logout", s.Logout)
 	s.router.GET("/transactions", authenticate, s.Transactions)
+	s.router.GET("/transactions-accept/:id", authenticate, s.TransactionsAccept)
```

Comment:
In REST parlance, "accept" is an action, so this URL should probably be:

```suggestion
	s.router.GET("/transactions/:id/accept", authenticate, s.TransactionsAcceptPreview)
```

I'm trying to keep things pretty formal since this code has an open source audience. Also we'll probably have the transactions detail page at `/transactions/:id` so this keeps things consistent.